Backend/app/user/user_service.py

```diff
@@ -201,6 +201,87 @@ def upgrade_to_regular_licence(
         except sqlite3.IntegrityError:
             return {"error": "No se pudo actualizar el usuario"}
 
+    # -------------------------------------------
+    def delete_user(self, user_email: str) -> User:
+        try:
+            # Lista de consultas para eliminar de cada tabla
+            consultas = [
+                f"DELETE FROM users WHERE email = ?",
+                f"DELETE FROM deviceRSAS WHERE email = ?",
+                f"DELETE FROM loan WHERE userEmail = ?",
```

Comment:
I think just deleting an user doesn't work. What if the user took out a loan and they didn't return it and then we delete it? We should add a "disabled" flag to users and prevent them from accessing loans or any other functionality but still keep a record of their loans.